dashboards/node.libsonnet

```
@@ -87,7 +88,7 @@ local gauge = promgrafonnet.gauge;
         legend_rightSide='true',
       ).addTarget(prometheus.target(
         |||
-          avg (sum by (cpu) (irate(node_cpu{%(nodeExporterSelector)s, mode!="idle", instance="$instance"}[2m])) ) * 100
+          max (sum by (cpu) (irate(node_cpu_seconds_total{%(nodeExporterSelector)s, mode!="idle", instance="$instance"}[2m])) ) * 10
```

This one is a bug and results in a flat graph.
```suggestion
          max (sum by (cpu) (irate(node_cpu_seconds_total{%(nodeExporterSelector)s, mode!="idle", instance="$instance"}[2m])) ) * 100
```